websites/N/Nekopost/dist/metadata.json

```diff
@@ -0,0 +1,22 @@
+{
+    "$schema": "https://schemas.premid.app/metadata/1.3",
+    "service": "B",
+    "author": {
+      "name": "B",
+      "id": "424565046748643328"
+    },
+    "category": "other",
+    "logo": "https://www.nekopost.net/demo/nekopost_logo.png",
+    "thumbnail": "https://www.nekopost.net/demo/nekopost_logo.png",

```

Comment:
Find the actual thumbnail, use the screenshot of home page if you can't find one, and upload to imgur